go/oasis-node/cmd/node/node_control.go

```diff
@@ -331,6 +331,20 @@ func (n *Node) getRuntimeStatus(ctx context.Context) (map[common.Namespace]contr
 			}
 		}
 
+		// Fetch provisioner type.
+		_, provisioner, err := rt.Host(ctx)
+		if err != nil {
+			n.logger.Error("failed to fetch host configuration",
+				"err", err,
+			)
+		} else {
+			if provisioner != nil {
+				status.Provisioner = provisioner.Name()
+			} else {
+				status.Provisioner = "none"
```

Comment:
```
		switch {
		case err != nil:
			n.logger.Error("failed to fetch host configuration",
				"err", err,
			)
		case provisioner != nil:
			status.Provisioner = provisioner.Name()
		default:
			status.Provisioner = "none"
		}
```
Could also use an empty string, as the comment says this is the name of the provisioner.

Reply:
I think "none" is a bit more user-friendly in this case :)